app/diagnostic/[sessionId]/page.tsx

```
@@ -249,172 +135,187 @@
+    const currentQuestion = sessionData.questions[currentQuestionIndex];
+    if (!currentQuestion) return;
+
+    const options = currentQuestion.options;
+    const currentIndex = selectedOptionLabel ? 
+      options.findIndex(opt => opt.label === selectedOptionLabel) : -1;
+
+    switch (event.key) {
+      case 'ArrowDown':
+      case 'j':
+        event.preventDefault();
+        const nextIndex = currentIndex < options.length - 1 ? currentIndex + 1 : 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)